server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/step/LoadReportAnalysisMetadataHolderStep.java

```diff
@@ -27,8 +27,8 @@
 import org.sonar.scanner.protocol.output.ScannerReport.Metadata.QProfile;
 import org.sonar.server.computation.task.projectanalysis.analysis.MutableAnalysisMetadataHolder;
 import org.sonar.server.computation.task.projectanalysis.batch.BatchReportReader;
-import org.sonar.server.computation.task.projectanalysis.qualityprofile.QualityProfile;
 import org.sonar.server.computation.task.step.ComputationStep;
+import org.sonar.server.qualityprofile.QualityProfile;
```

Comment:
It was in CE because it was only used in CE, but I need exactly the same class, some I've moved it in a common package